src/execution_plan/record.c

```diff
@@ -78,7 +78,7 @@ RecordEntryType Record_GetType(const Record r, int idx) {
 
 SIValue Record_GetScalar(Record r,  int idx) {
 	r[idx].type = REC_TYPE_SCALAR;
-	return SI_MakeVolatile(r[idx].value.s);
+	return r[idx].value.s;
```

Comment:
Is it safe to simply return this scalar as is?

Reply:
is the question still valid?